sys-cluster/keepalived/keepalived-2.0.6.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="A strong & robust keepalive facility to the Linux Virtual Server project"
+HOMEPAGE="http://www.keepalived.org/"
+SRC_URI="http://www.keepalived.org/software/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sparc ~x86"
+IUSE="dbus debug ipv6 -json snmp"
+
+RDEPEND="dev-libs/libnl:=
+	dev-libs/openssl:=
+	dev-libs/popt
+	net-libs/libnfnetlink
+	sys-apps/iproute2
+	dbus? ( sys-apps/dbus )
+	json? ( dev-libs/json-c:= )
+	snmp? ( net-analyzer/net-snmp )"
+DEPEND="${RDEPEND}
+	>=sys-kernel/linux-headers-4.4"
+
+DOCS=(
+	README CONTRIBUTORS INSTALL ChangeLog AUTHOR TODO
+	doc/keepalived.conf.SYNOPSIS doc/NOTE_vrrp_vmac.txt
+)
+
+src_prepare() {
+	default
+
+	eautoreconf
+}
+
+src_configure() {
+	STRIP=/bin/true \
+	econf \
+		--with-kernel-dir=/usr \
+		--enable-sha1 \
+		--enable-vrrp \
+		$(use_enable dbus) \
+		$(use_enable dbus dbus-create-instance) \
+		$(use_enable debug) \
+		$(use_enable json) \
+		$(use_enable snmp)
+}
+
+src_install() {
+	default
+
+	newinitd "${FILESDIR}"/keepalived.init keepalived
+	newconfd "${FILESDIR}"/keepalived.confd keepalived
+
+	use snmp && dodoc doc/KEEPALIVED-MIB.txt
+
+	docinto genhash
+	dodoc genhash/README genhash/AUTHOR genhash/ChangeLog
```

Comment:
This a minor nitpick. Could you add those files to the `DOCS` array and get rid of the `dodoc` call? Thanks!

Reply:
Since the genhash filenames collide with keepalived, it would probably make a mess :(

It looks like this now:
/usr/share/doc/keepalived-2.0.6/AUTHOR
/usr/share/doc/keepalived-2.0.6/CONTRIBUTORS.bz2
/usr/share/doc/keepalived-2.0.6/ChangeLog.bz2
/usr/share/doc/keepalived-2.0.6/INSTALL.bz2
/usr/share/doc/keepalived-2.0.6/NOTE_vrrp_vmac.txt.bz2
/usr/share/doc/keepalived-2.0.6/README.bz2
/usr/share/doc/keepalived-2.0.6/TODO.bz2
/usr/share/doc/keepalived-2.0.6/keepalived.conf.SYNOPSIS.bz2

/usr/share/doc/keepalived-2.0.6/genhash
/usr/share/doc/keepalived-2.0.6/genhash/AUTHOR
/usr/share/doc/keepalived-2.0.6/genhash/ChangeLog.bz2
/usr/share/doc/keepalived-2.0.6/genhash/INSTALL+HOWTO.bz2
/usr/share/doc/keepalived-2.0.6/genhash/README.bz2